.github/workflows/rotki_nightly_builds.yml

```diff
@@ -220,8 +220,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+      - name: Available platforms
+        run: echo ${{ steps.buildx.outputs.platforms }}
```

Comment:
This doesn't seem to print anything so maybe we could remove it?

Reply:
Indeed ! I'll remove them